TestCases/incomp_navierstokes/streamwise_periodic/chtPinArray_2d/of_grad_findiff.csv.ref

```diff
@@ -1,2 +1,2 @@
 "VARIABLE"      , "AVG_DENSITY[0]", "AVG_ENTHALPY[0]", "AVG_NORMALVEL[0]", "DRAG[0]"       , "EFFICIENCY[0]" , "FORCE_X[0]"    , "FORCE_Y[0]"    , "FORCE_Z[0]"    , "LIFT[0]"       , "MOMENT_X[0]"   , "MOMENT_Y[0]"   , "MOMENT_Z[0]"   , "SIDEFORCE[0]"  , "SURFACE_MACH[0]", "SURFACE_MASSFLOW[0]", "SURFACE_MOM_DISTORTION[0]", "SURFACE_PRESSURE_DROP[0]", "SURFACE_SECONDARY[0]", "SURFACE_SECOND_OVER_UNIFORM[0]", "SURFACE_STATIC_PRESSURE[0]", "SURFACE_STATIC_TEMPERATURE[0]", "SURFACE_TOTAL_PRESSURE[0]", "SURFACE_TOTAL_TEMPERATURE[0]", "SURFACE_UNIFORMITY[0]", "AVG_TEMPERATURE[1]", "MAXIMUM_HEATFLUX[1]", "TOTAL_HEATFLUX[1]", "FINDIFF_STEP"  
-0               , 0.0             , -99999.96982514858, 2.2204999999731917e-08, 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , -0.04999999997368221, -1.1100000001884e-08, -2.069999999187999       , 0.0                     , 2.12000000054946    , 3.7100000016554446            , 330.00000030369847        , -39.999997625272954          , 315.0000011942211        , -40.000008993956726         , -1.400000004814217   , -149.9999996212864, 0.0                , -469.9999976764957, 1e-08           
+0               , 0.0             , -99999.96982514858, 3.330700000025998e-08, 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , 0.0             , -0.04999999997368221, -1.1100000001884e-08, -2.069999999187999       , 0.0                     , 2.12000000054946    , 3.7100000016554446            , 330.00000030369847        , -39.999997625272954          , 315.0000011942211        , -40.000008993956726         , -1.400000004814217   , -149.9999996212864, 0.0                , -469.9999976764957, 1e-08           
```

Comment:
🔎 what's going on here? (breakfast 🐞  obviously, but jokes aside) was this a bug? I guess it is 1e-8 land ...

Reply:
My different handling of division by zero maybe, or the optimizability of 2 or 3

Reply:
Can you give me place to look for the different handling of division by zero 🔬 (so I dont have to think for myself)